docs/admin/kubernetes-ops/custom-attributes.md

```diff
@@ -0,0 +1,116 @@
+---
+tags:
+  - administration
+  - kubernetes
+  - operations
+  - custom attributes
+  - schema
+  - claim
+---
+
+
+## Overview
+Making changes in the schema to add custom columns/attributes
+
+1.  Create a directory and download the needed files inside it
+
+    ```
+    mkdir schema && cd schema
+    wget https://raw.githubusercontent.com/JanssenProject/jans/main/jans-linux-setup/jans_setup/schema/jans_schema.json
```

Comment:
Any reason why we need to download and mount unmodified `jans_schema.json`?

Reply:
> Any reason why we need to download and mount unmodified `jans_schema.json`?
If we didn't, it gives the following error
![Screenshot (930)](https://github.com/JanssenProject/jans/assets/49232835/2bed0292-5443-4c32-92be-43570c0716a4)

Reply:
Have you tried mounting `custom_schema.json` using `subPath` like this:

![image](https://github.com/JanssenProject/jans/assets/231601/2dbae485-6592-43b4-86eb-558a89f436d8)